includes/wikia/tests/UserPasswordTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+use Wikia\Service\Helios\ClientException;
+use Wikia\Service\Helios\HeliosClient;
+use Wikia\DependencyInjection\Injector;
+
+class UserPasswordTest extends WikiaBaseTest {
+
+	const TEST_USER_ID = 4;
+
+	/** @var User */
+	protected $testUser;
+
+	protected static $currentInjector;
+
+	public static function setUpBeforeClass() {
+		self::$currentInjector = Injector::getInjector();
+	}
+
+	public function setUp() {
+		parent::setUp();
+		$this->testUser = User::newFromId( self::TEST_USER_ID );
+	}
+
+	public static function tearDownAfterClass() {
+		Injector::setInjector( self::$currentInjector );
+	}
+
+	public function testShouldSetNewPassword() {
+		$this->assertTrue( $this->testUser->setPassword( 'goodpassword123' ) );
+	}
+
+	// @todo uncomment once Helios supports password deletion
+//	public function testShouldDeletePassword() {
+//		$password = "fhsdakljhasfdhjjfdjh2345";
+//		$this->testUser->setPassword( $password );
+//		$this->assertTrue( $this->testUser->setPassword( null ) );
+//		$this->assertEquals(
+//			'401',
+//			Injector::getInjector()->get( HeliosClient::class )
+//				->login( $this->testUser->getName(), $password )[0]
+//		);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)